exercises/concept/interest-is-interesting/.docs/introduction.md

```diff
@@ -10,13 +10,13 @@ C# has three floating-point types:
 
 - `float`: 4 bytes (~6-9 digits precision). Written as `2.45f`.
 - `double`: 8 bytes (~15-17 digits precision). This is the most common type. Written as `2.45` or `2.45d`.
-- `decimal`: 16 bytes (28-29 digits precision). Normally used when working with monetary data, as its precision leads to less rounding errors. Written as `2.45m`.
+- `decimal`: 16 bytes (28-29 digits precision). Normally used when working with monetary data, as its precision reduces the chance of rounding errors. Written as `2.45m`.
 
-As can be seen, each type can store a different number of digits. This means that trying to store PI in a `float` will only store the first 6 to 9 digits (with the last digit being rounded).
+As can be seen, each type can store a different number of digits. Trying to store PI in a `float` will only store the first 6 decimal places (with the last stored digit rounded).
```

Comment:
I think this sentence should be linked to the previous. 

```suggestion
As can be seen, each type can store a different number of digits. For example, trying to store PI in a `float` will only store the first 6 decimal places (with the last stored digit rounded).
```

Reply:
Makes sense, thanks for the suggestion!